.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*' # older version branches
+    tags:
+      - '*'
+  pull_request: {}
+  schedule:
+  - cron:  '0 6 * * 0' # weekly, on sundays
```

Comment:
FWIW I find a nightly run that logs an issue to be nicer than just running the action
https://github.com/hjdivad/ember-m3/blob/master/.github/workflows/nightly.yml

Reply:
Thats a great idea, I'll take a look at that and add it to my GH template.